### ISSUE TITLE
Update builder documentation to include the missing (boot) `firmware` parameter

### ISSUE THREAD

**Issue by justinkarabegovic** [ddelnano/packer-plugin-xenserver#69]
Please add the option to select between `bios` and `uefi` boot firmware for the VM.

This could initially be added as an option to `platform_args`. This could then be migrated to a standalone argument later.

Perhaps could be implemented in:

File: `https://github.com/ddelnano/packer-plugin-xenserver/blob/master/builder/xenserver/iso/builder.go`

Line 84:
```
	if len(self.config.PlatformArgs) == 0 {
		pargs := make(map[string]string)
		pargs["viridian"] = "false"
		pargs["nx"] = "true"
		pargs["pae"] = "true"
		pargs["apic"] = "true"
		pargs["timeoffset"] = "0"
		pargs["acpi"] = "1"
		self.config.PlatformArgs = pargs
	}
```

add an additional option such as:
```
		pargs["hvm_boot_firmware"] = "bios"
```
Which would match parameter in `https://github.com/terra-farm/terraform-provider-xenorchestra/blob/master/xoa/resource_xenorchestra_vm.go`

**Comment by ddelnano** [comment]
@justinkarabegovic this is already possible in the latest release and was implemented in #39. I think the confusion here is that we forgot to update the docs to include that parameter.

**Comment by ddelnano** [comment]
Please let me know if you run into any issues using it, but I believe this is complete now.
